client/src/components/Header.js

```diff
@@ -98,7 +99,38 @@ const HeaderWrapper = styled.div`
   width: 100vw;
 `;
 
-export default ({ authLoading, onMenuClick, isAuthenticated }) => {
+export default ({ session, authLoading, onMenuClick, isAuthenticated, orgname, user}) => {
```

Comment:
Also, you are not using session, and orgname here. Where are you using these ?

Reply:
Good catch ! I was actually testing out passing the data from one component to another to view the organisation names. Will remove now